sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/client/HttpClient.java

```diff
@@ -20,11 +21,31 @@ public interface HttpClient {
     Response<?> send(HttpRequest request);
 
     /**
-     * Creates a new {@link HttpClient} instance.
+     * Get a new instance of the {@link HttpClient} that the default {@link HttpClientProvider} is configured to
```

Comment:
```suggestion
     * Get a new instance of the {@link HttpClient} that the {@link HttpClientProvider} is configured to
```
default could be misleading here.